README.md

```diff
@@ -264,6 +264,15 @@ specified first. Also, `--check` ignores the value of the `controller` and assum
 will be present during a real run. That means, in presence of an invalid `controller`,
 `--check` may signal success but the actual play run fails.
 
+When destroying and removing the bonding profiles, if only specifying the `controller`
+profile in the `connections` list and bringing it down, the port profiles will be
+brought down automatically. However, if not all the port profiles are brought down,
+the connection state of the `controller` profile will not be affected (stay `up`).
```

Comment:
This part is quite confusing for me. Could you separate the 3 cases?
1. Removing only the controller.
2. Removing all the ports.
3. Removing some ports.

Reply:
Also, this only needs to be documented, if there is something special that users need to consider/could be surprising. If it basically works in any order, then it might not need to be documented in such detail.

Reply:
In terms of removing all the ports, there is some difference between the nm provider and initscripts provider. For nm provider, if removing all the ports, we can still have the active controller connection. But for initscripts, if removing all the ports, the controller will also be brought down.

Reply:
The README and the tests have been updated. Thanks.